update_metadata_bdc/v1.0.0/Dockerfile

```diff
@@ -39,9 +40,8 @@ ENV R_VERSION 4.3.2
 
 # Configure CRAN for package retrieval
 RUN echo "r <- getOption('repos'); r['CRAN'] <- 'http://cran.us.r-project.org'; options(repos = r);" > ~/.Rprofile
-RUN Rscript -e "install.packages('getopt', dependencies = T)"
-RUN Rscript -e "install.packages(c('dplyr', 'httr', 'stringr','lubridate','sevenbridges2'), dependencies = T)"
-RUN Rscript -e "library('getopt');##### R SESSION INFORMATION #####; sessionInfo()"
+RUN Rscript -e "install.packages(c('getopt', 'dplyr', 'httr', 'stringr','lubridate','sevenbridges2'), dependencies = T)"
+RUN Rscript -e "library('getopt'); ##### R SESSION INFORMATION #####; sessionInfo()"
```

Comment:
* Why is `getopt` installed twice?
* is there a reason why this is all on one line? If not, please put the comment and `sessionInfo()` command on a new line.